src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -2060,6 +2067,47 @@ protected virtual DiagnosticInfo GetUseSiteDiagnosticImpl()
             return diagnostic;
         }
 
+        protected override int HighestPriorityUseSiteError => (int)ErrorCode.ERR_UnsupportedCompilerFeature;
```

Comment:
>protected override int HighestPriorityUseSiteError => (int)ErrorCode.ERR_UnsupportedCompilerFeature; [](http://example.com/codeflow?start=8&length=100)

This change makes me pause. ```ErrorCode.ERR_BogusType``` used to have highest priority. Now it doesn't and ```ERR_UnsupportedCompilerFeature``` won't be common. Effectively, it looks like we are removing prioritization.

Is this override really necessary, given that ```GetUseSiteDiagnosticImpl``` prioritizes ```ErrorCode.ERR_UnsupportedCompilerFeature``` explicitly?

Reply:
Do we want to have more complicated priority scheme now? With two high pri codes?